src/amo/reducers/blocks.js

```diff
@@ -15,6 +15,8 @@ export type ExternalBlockType = {|
   guid: string,
   id: number,
   versions: [string],
```

Comment:
The API still returns it though, so I figured I might as well leave them in the type

Reply:
the API won't once the gate is disabled for it, and it - it's only enabled right now (and [undocumented](https://mozilla.github.io/addons-server/topics/api/blocklist.html)) because frontend needs it until this patch lands.